dashboard/app/controllers/sessions_controller.rb

```diff
@@ -56,6 +56,27 @@ def reset
     render layout: false
   end
 
+  # GET /lockout
+  # This page is for accounts that are locked until parental permission compliance.
+  def lockout
+    # Basic defaults. If the @pending_email is empty, the request was never sent
+    @pending_email = ""
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby-string-quotes

Reply:
interesting that the linter did not catch this. a common thing I do wrong. good call.